### PR TITLE
feat(recovery-lab): update content

### LINKED ISSUE #8: Recover from blocks.log, snapshot or backup. (recovery-lab)
@leisterfrancisco maybe we can ask the student to run nodeos and then shutdown it ungracefully, e.g. with `kill -kill $(pidof nodeos)`. And then recover from blocks.log, snapshot or backup.

_Originally posted by @kuronosec in https://github.com/eoscostarica/sample-nodeos-configs/pull/7#discussion_r777706711_

### PR DESCRIPTION
**Pull request by leisterfrancisco**
## Update recovery-lab content using snapshot

Resolved #8 

The student will need to edit the `config.ini` file created in [Run a Local Producer Node](https://github.com/eoscostarica/sample-nodeos-configs/blob/main/producer-node-lab.md) laboratory by adding the `producer_api_plugin` plugin. Later create the snapshot and replay `nodeos` from it.

### CODE REVIEW

**Review by xavier506 — APPROVED** [review]
LGTM

File: recovery-lab.md [hunk 1]
```diff
@@ -1,34 +1,144 @@
...
+> This laboratory assumes the project is running in your local machine
 
-Although rare, emergency software releases may occur that require compiling a patched version of EOSIO. In these situations it is important that node operators respond quickly in applying the updates to ensure the security and stability of the network. 
+The student must send:
+1. The `nodeos.log` file created in the step `Finally, run nodeos`.
 
-We have provided you with a sample list of commands in order to patch EOSIO if this scenario occurs: 
+### Help material
+
+To help you complete this tutorial, you can consult next material:
```
Comment (kuronosec) [inline recovery-lab.md:18]: "..., you can take a look of the following resource:"

File: recovery-lab.md [hunk 2]
```diff
@@ -1,34 +1,144 @@
...
+- [Nodeos Replays](https://developers.eos.io/manuals/eos/v2.0/nodeos/replays/index).
+
+## Prerequisite
+To successfully complete this laboratory, please, make sure you have next requirement:
```
Comment (kuronosec) [inline recovery-lab.md:22]: "... you have *the* next..."

File: recovery-lab.md [hunk 3]
```diff
@@ -1,34 +1,144 @@
...
+enable-stale-production = true
+producer-name = eosio
+plugin = eosio::producer_plugin
+
+# New plugin to create snapshots
+plugin = eosio::producer_api_plugin
+```
+
+### 2. Generate a Snapshot
+To restore a dawn node or start a new one from a snapshot, firstly it is needed to have created the snapshot file. To do so, run:
```
Comment (kuronosec) [inline recovery-lab.md:43]: "stoped node". "...first you need a snapshot..."

File: recovery-lab.md [hunk 4]
```diff
@@ -1,34 +1,144 @@
...
+### 3. Replay from a Snapshot
+For this laboratory, we are going to replay from an empty folder, follow next steps to successfully replay nodeos from a snapshot.
+
+1. Run the following command to create the working folder.
+
+```sh
+mkdir ~/recovery-lab
+```
+
+2. Copy the snapshot you created in [step two](https://github.com/eoscostarica/sample-nodeos-configs/blob/main/recovery-lab.md#2-generate-a-snapshot) to the working folder.
```
Comment (kuronosec) [inline recovery-lab.md:61]: working directory

File: recovery-lab.md [hunk 5]
```diff
@@ -1,34 +1,144 @@
...
+
+```sh
+cp <SNAPSHOT_NAME>.bin ~/recovery-lab
+```
+
+3. Create next variables in your terminal:
```
Comment (kuronosec) [inline recovery-lab.md:67]: "...create *the* following..."

File: recovery-lab.md [hunk 6]
```diff
@@ -1,34 +1,144 @@
...
+
+```sh
+SNAPSHOT=latest.bin
+CONFIG_DIR=./config
+DATA_DIR=./blockchain
+```
+
+4. Rename the snapshot file for a better comprehension:
```
Comment (kuronosec) [inline recovery-lab.md:75]: for simplicity

File: recovery-lab.md [hunk 7]
```diff
@@ -1,34 +1,144 @@
...
+> In case you set a different name than `latest.bin` for the snapshot file, make sure to update the value for `SNAPSHOT` variable to the new name.
+
+5. Create the snapshot folder.
+
+```sh
+cd ~/recovery-lab
+mkdir -p $DATA_DIR/snapshots
+```
+
+6. Copy the snapshot file located in your working folder to the snapshots folder.
```
Comment (kuronosec) [inline recovery-lab.md:91]: working directory

**Pull request merged**